hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/DomainBuilder.java

```diff
@@ -473,6 +479,10 @@ public long id() {
         return id.incrementAndGet();
     }
 
+    public int transactionIndex() {
+        return transactionIndex.incrementAndGet();
```

Comment:
nit, since the index starts from 0 should do:

```suggestion
        return transactionIndex.getAndIncrement();
```

Reply:
Fixed